features/atcryptoauth/ATCADevice.h

```diff
@@ -184,8 +211,18 @@ class ATCADevice  : public CryptoEngineInterface
     ATCAError   Wakeup();
     /** Reset watchdog timer.
      *
-     *  Avoid deep sleep and loss of current command state on watchdog timer
-     *  expiry by putting device in Idle mode and straight away waking up.
+     *  Device goes to sleep on Twatchdog expiry. This driver uses default
```

Comment:
Stray letter in “watchdog”.

Reply:
```Twatchdog``` is the timer name in the datasheet. I can omit ```T``` if you want.

Reply:
Oh I hadn't realized this was the Twatchdog variable and not the English word watchdog. Ignore my comment then.